go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -466,6 +466,229 @@ func TestSystemTableQueries(t *testing.T) {
 	enginetest.RunQueryTests(t, newDoltHarness(t), BrokenSystemTableQueries)
 }
 
+func TestUnscopedDoltDiffSystemTable(t *testing.T) {
+	// There's a bug in queries with where clauses that compare column equality with a
+	// variable. These tests use "commit_hash in (@Commit1)" to work around that bug.
+	// https://github.com/dolthub/go-mysql-server/issues/790
+
+	var tests = []enginetest.ScriptTest{
```

Comment:
I wanted to pull these out into a separate file... is extracting the []ScriptTest var into a separate file the preferred pattern?